README.md

```diff
@@ -1,22 +1,50 @@
 
 ## Architecture
 
-**This architecture is infrastructure focused**, more so than on workload. It concentrates on the AKS cluster itself, including concerns with identity, post-deployment configuration, secret management, and network topologies.
-
-The implementation presented here is the _minimum recommended baseline for most AKS clusters_. This implementation integrates with Azure services that will deliver observability, provide a network topology that will support multi-regional growth, and keep the in-cluster traffic secure as well. This architecture should be considered your starting point for pre-production and production stages.
+**This architecture is infrastructure focused again**, more so than on workload. It concentrates on two AKS clusters, including concerns like multi region deployments, the desired state of the cluster, geo replication, network topologies and more.
 
-The material here is relatively dense. We strongly encourage you to dedicate time to walk through these instructions, with a mind to learning. We do NOT provide any "one click" deployment here. However, once you've understood the components involved and identified the shared responsibilities between your team and your great organization, it is encouraged that you build suitable, auditable deployment processes around your final infrastructure.
+The implementation presented here like in the baseline is the _minimum recommended starting (baseline) for a multiple AKS cluster solution_. This implementation integrates with Azure services that will deliver geo replication, a centralized observability appraoch, a network topology that is going go with multi-regional growth for real, and balancing the all traffic as well.
 
-Throughout the reference implementation, you will see reference to _Contoso Bicycle_. They are a fictional small and fast-growing startup that provides online web services to its clientele on the west coast of North America. They have no on-premises data centers and all their containerized line of business applications are now about to be orchestrated by secure, enterprise-ready AKS clusters. You can read more about [their requirements and their IT team composition](./contoso-bicycle/README.md). This narrative provides grounding for some implementation details, naming conventions, etc. You should adapt as you see fit.
+The material here will try not to be dense or at least focused exclusevely on the multi regional growth. We strongly encourage you to allocate time to walk through the AKS Basline and later all the instructions provided in here, with a mind to learning. We do NOT provide any "one click" deployment here. However, once you've understood the components involved and identified the shared responsibilities between your team and your great organization, it is encouraged that you build suitable, auditable deployment processes around your final infrastructure.
```

Comment:
> The material here will try not to be dense or at least focused exclusively on the multi-regional growth

Not sure what that sentence is really trying to say.

Reply:
tried to clarify a bit the intent from [c36a4](https://github.com/mspnp/aks-secure-baseline/pull/174/commits/c36a4695126e483a607c8684d2c9f9f1d6b621e5)